### PR TITLE
Fix typos in bad test README

### PR DESCRIPTION
**Pull request by PeytonT**
*Issue #, if available:*

*Description of changes:* Spotted some misc typos while looking at bad binary Decimal tests.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: iontestdata/bad/README.md [hunk 1]
```diff
@@ -65,18 +65,18 @@ data are available.
 
 decimalExpTooLarge.10n
 ----------------------
-This file contains a decimal who's exponent exceeds the length defined by the
+This file contains a Decimal who's exponent exceeds the length defined by the
```
Comment (tgregg) [inline iontestdata/bad/README.md]: While you're here, would you mind correcting "who's" -> "whose"?

File: iontestdata/bad/README.md [hunk 2]
```diff
@@ -65,18 +65,18 @@ data are available.
...
 decimal container's length.
 
 decimalLenCauses64BitOverflow.10n
 ---------------------------------
-This file contains a decimal who's total length is 2^64-1, larger than the
+This file contains a Decimal who's total length is 2^64-1, larger than the
```
Comment (tgregg) [inline iontestdata/bad/README.md]: While you're here, would you mind correcting "who's" -> "whose"?

File: iontestdata/bad/README.md (hunk 1 above)
Comment (PeytonT) [reply iontestdata/bad/README.md]: Done

File: iontestdata/bad/README.md (hunk 2 above)
Comment (PeytonT) [reply iontestdata/bad/README.md]: Done

**Comment by tgregg** [comment]
@PeytonT Thanks!

**Pull request merged**